modules/ROOT/pages/import-mule-app-project.adoc

```diff
@@ -12,6 +12,15 @@ You can import a Mule-application project that was exported from Design Center.
 * Importing a project that was exported from a different environment within a single organization
 * Importing a project that was exported from an environment that is in a different organization
 
+[IMPORTANT]
+====
+An exported project does not include items that the project linked to in its original location.
+
+For example, if the project contains a Database connector, the database driver for that connector is not included in the exported project. After you import the project, you must open the Database connector, browse to the location of the database driver, and select it.
+
+If the project contains any connectors that require a driver or another type of library, you must open those connectors and select the required libraries.
```

Comment:
I would rephrase this to 'you must open those connector configurations and set up the required libraries' or something similar. The thing I want to emphasize is that drivers and libraries are normally set up in the connector reusable configurations.